src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/UseEnvironmentProcessIdTests.cs

```diff
@@ -79,27 +107,28 @@ public static class Environment
 
 class C
 {
-    int M()
+    int HandleProcessId()
     {
-        int pid = [|Process.GetCurrentProcess().Id|];
-        pid = [|Process.GetCurrentProcess()/*willberemoved*/.Id|];
-        Use([|Process.GetCurrentProcess().Id|]);
+        int pid = {|CA1837:Process.GetCurrentProcess().Id|};
```

Comment:
Should we have tests for something like:

```csharp
if (Process.GetCurrentProcess() is { Id: some_value })
{
    // do something
}
```

Reply:
We could, but I don't think it's necessary.  There are lots of different ways to express the same thing, and all we really care about right now is the one that we've actually seen devs write.

Reply:
@stephentoub I didn't mean to necessarily support this, but at least to make sure that this doesn't cause the analyzer or fixer to crash.